Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
```

Comment:
It would be even clearer to not say anything ;)

The `binary:` target could contain something like `$(if DEBUG=1,DEBUG=1,)` (untested) to do that, but that’s getting too complicated for just passing a single boolean down (for my, entirely subjective and perhaps weird and wrong, taste).